src/client.rs

```diff
@@ -297,6 +298,36 @@ pub extern "C" fn rustls_client_config_builder_load_roots_from_file(
     }
 }
 
+/// Set the ALPN protocol list to the given protocols. `protocols` must point
+/// to a buffer of `rustls_slice_bytes` (built by the caller) with `len`
+/// elements. Each element of the buffer must point to a slice of bytes that
+/// contains a single ALPN protocol from
+/// https://www.iana.org/assignments/tls-extensiontype-values/tls-extensiontype-values.xhtml#alpn-protocol-ids.
```

Comment:
rustls doesn't enforce any restriction like that, so maybe this should be removed/worded differently?

Reply:
The restriction being "contains ... from <url>"? How about:

Each element of the buffer must point to a slice of bytes containing a single ALPN protocol. Standard ALPN protocols are registered at <url>.